examples/java/src/test/java/org/apache/beam/examples/WindowedWordCountIT.java

```diff
@@ -67,6 +68,14 @@ private void testWindowedWordCountPipeline(boolean isStreaming) throws IOExcepti
         TestPipeline.testingPipelineOptions().as(WindowedWordCountITOptions.class);
     options.setStreaming(isStreaming);
 
+    // Note: currently unused because the example writes to BigQuery, but WindowedWordCount.Options
+    // are tightly coupled to WordCount.Options, where the option is required.
+    options.setOutput(IOChannelUtils.resolve(
```

Comment:
I don't have a strong opinion at this time. This replicates the current practice from `WordCountIT`:

https://github.com/apache/incubator-beam/blob/master/examples/java/src/test/java/org/apache/beam/examples/WordCountIT.java#L59